usr/sbin/p11sak/p11sak.c

```diff
@@ -2216,8 +2225,9 @@ static CK_RV build_cmd_opts(const struct p11sak_opt *cmd_opts,
     CK_RV rc;
 
     count_opts(p11sak_generic_opts, &optstring_len, &longopts_count);
-    if (cmd_opts != NULL)
+    if (cmd_opts != NULL) {
```

Comment:
I guess this is an unrelated change, or a left over from a earlier stage. 
For a single line body, no  { } are needed.